core/src/main/scala/io/github/rafafrdz/criteria4s/core/Show.scala

```diff
@@ -0,0 +1,18 @@
+package io.github.rafafrdz.criteria4s.core
+
+trait Show[-V, D <: CriteriaTag] {
+  def show(v: V): String
+}
+
+object Show {
+  def create[D <: CriteriaTag]: ShowBuilder[D] = new ShowBuilder[D]
+
+  private[core] final class ShowBuilder[D <: CriteriaTag] {
+    def apply[V](f: V => String): Show[V, D] = (v: V) => f(v)
+  }
+
+  type ShowColumn[D <: CriteriaTag] = Show[Column, D]
+
+  implicit def defaultStringShow[D <: CriteriaTag]: Show[String, D] = identity
+  implicit def defaultIntShow[D <: CriteriaTag]: Show[Int, D] = create(_.toString)
+}
```

Comment:
I don't know why did you do the `ShowBuilder`, you can just use a smart constructor in order to create a new instance of `Show`.

On the other hand, either String type or Int type has a default implicit show instance by using identity or _.toString, it would be nice use _.toString for all types as a default instance for show 😀.

And finally, it is just a personal opinion, but I prefer to not have a particular type alias for ShowColumn, use just `Show[Column, D]` like the rest of the types. I mean, for all types they will be typed as `Show[SomeType, D]` except Column? But again for is a personal opinion. 😅

```suggestion
object Show {

  def create[D <: CriteriaTag, V](f: V => String): Show[V, D] = (v: V) => f(v)

  implicit def defaultShow[D <: CriteriaTag, V]: Show[V, D] = create(v => v.toString)
}

```